docs/couchdb.md

```diff
@@ -0,0 +1,50 @@
+# `couchdb` Metrics Receiver
+
+The couchdb receiver can retrieve stats from your couchdb server using the `/_node/_local/_stats/couchdb` endpoint.
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your Apache web server configuration.
+
+To configure a receiver for your Apache web server metrics, specify the following fields:
+
+| Field                 | Default                 | Description                                                                            |
+|-----------------------|-------------------------|----------------------------------------------------------------------------------------|
+| `type`                | required                | Must be `couchdb`.                                                                     |
+| `endpoint`            | `http://localhost:5984` | The url exposed by couchdb                                                             |
+| `username`            | not set by default      | The username used to connect to the server.                                            |
+| `password`            | not set by default      | The password used to connect to the server.                                            |
+| `collection_interval` | `60s`                   | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    couchdb_metrics:
+      type: couchdb
+      endpoint: http://localhost:5984
+      collection_interval: 60s
```

Comment:
`collection_interval` is no longer a required field. We can remove it from the sample config.

Reply:
updated